README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+Biography
+=========
+
+Born and raised in South Korea, high school in America, rising third
+year undergrad majoring in math and economics. Classical musician
+(trying to be an adequate one). Likes reading and running.
+
+-   Favorite things
+    -   Composer: **J.S. Bach**
+    -   Conductor: Wilhelm Furtwangler
+    -   Pianist: Sviatoslav Richter, Wilhelm Kempff, Glenn Gould
+    -   Violinist: Jascha Heifetz, David Oistrakh, Nathan Milstein,
+        Arthur Grumiaux
+
+![Portrait of J.S.Bach](https://upload.wikimedia.org/wikipedia/commons/thumb/6/6a/Johann_Sebastian_Bach.jpg/220px-Johann_Sebastian_Bach.jpg)
+
+Progress report
+===============
+
+Opened README.md with R Studio, made appropriate changes, specified md

```

Comment:
Nice description! Next time, include this text in your pull request as a comment, rather than in the repo itself